scraper.rb

```diff
@@ -16,6 +18,8 @@ def scrape(url)
 
 end
 
+OCD_MAPPING = csv = CSV.parse(open('https://morph.io/chrismytton/uganda_ocd_ids/data.csv?key=e2EFi0ZRV56uq%2BtrzZ0H&query=select+%2A+from+%27data%27').read, headers: true, header_converters: :symbol).map(&:to_hash)
```

Comment:
Why the double assignment here? I don't think `csv` actually gets used again, does it?